phpstan-dbal2.neon

```diff
@@ -3,6 +3,8 @@ includes:
     - phpstan-params.neon
 
 parameters:
+    reportUnmatchedIgnoredErrors: false
```

Comment:
For the record: I don't care about unmatched baseline errors when analyzing with DBAL 2. It's a waste of time. 😎